example/ios/Runner/Info.plist

```diff
@@ -14,8 +14,6 @@
 	<string>6.0</string>
 	<key>CFBundleName</key>
 	<string>com.fluttercandies.photo_manager_example</string>
-	<key>CFBundleDisplayName</key>
-	<string>photo_manager Example</string>
```

Comment:
Why is this being deleting?

Reply:
<img width="1646" alt="image" src="https://github.com/fluttercandies/flutter_photo_manager/assets/14145407/7e022dd5-4887-4b5d-ae13-b3d7887ac737">

I don't seem to have manually removed it.
I guest the xcode delete it when compile, because it repeats.